src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -953,6 +954,11 @@ def enable_streaming_for_llm_flow(self, stream_required: Callable[[], bool]):
 
         :return: None
         """
+        from promptflow.executor._script_executor import ScriptExecutor
```

Comment:
I think we can directly update _script_executor.py:
```
class ScriptExecutor:
    def enable_streaming_for_llm_flow(self, stream_required):
        return
```

Reply:
Sure, updated